src/ParquetMsg.chpl

```diff
@@ -417,6 +419,7 @@ module ParquetMsg {
                                         dtype, compression,
                                         errMsg): int;
     var dtypeRep = toCDtype(dtype);
+    var doParallel = if A.size > parallelWriteThreshold then true else false;
```

Comment:
okay this is pedantic, but this could just be `var doParallel = A.size > parallelWriteThreshold;` right?